distribution/lib/Standard/Database/0.0.0-dev/src/Feature.enso

```diff
@@ -41,8 +41,8 @@ type Feature
        currently blocks getting a DB_Column from a DB_Table, but will soon refine to operations on DB_Column.
     Column_Operations
     ## PRIVATE
-       set operations on tables.
-    Set
```

Comment:
Why is this flag no longer needed?

Reply:
Set is a fairly basic operation that all dbs should be able to support from day 1. The only hard bit comes from all of the operations and we have a separate flagging system for these. is_operation_supported